src/walletd/runtime.rs

```diff
@@ -942,6 +942,7 @@ impl Runtime {
                 accordant_addr,
                 taker_commit,
             }) => {
+                info!("{} | Received TakerCommitted", swap_id.swap_id());
```

Comment:
Maybe we should prefer plain English for info level logs (but not sure it is applied everywhere yet)

Reply:
1d1ec6ec54d8ce2dcd74a0f8e909047e10c03eed